### PR TITLE
docs(CIP-14): define proposed param values

### PR DESCRIPTION
**Pull request by rootulp**
Currently CIP-14 is ambiguous with respect to the values it proposes for the host module parameters. The table in this PR aims to remove that ambiguity.

The values were chosen based on the latest messages in the [forum post](https://forum.celestia.org/t/moving-toward-safer-and-more-aligned-tia-liquid-staking/1422/18). I would appreciate reviews from the CIP authors: @womensrights @asalzmann @sampocs 

Note: I think we also need to fill out the **Test Cases** section of this CIP prior to moving it to review

### CODE REVIEW

**Comment by jcstein** [comment]
Per CIP-1, test cases is optional, but for this CIP, I think it is helpful https://github.com/celestiaorg/CIPs/blob/main/cips/cip-template.md#test-cases

File: cips/cip-14.md
Comment (ebuchman) [inline cips/cip-14.md:38]: Thanks for the table!

> False

Should this be False? Potentially there may be desire to add more in the future?

File: cips/cip-14.md
Comment (ebuchman) [inline cips/cip-14.md:42]: Just realized a lot of this text is copied verbatim from the ICS27 spec in https://github.com/cosmos/ibc/tree/main/spec/app/ics-027-interchain-accounts . I think its good to have it here but maybe worth noting its a direct copy?

File: cips/cip-14.md
Comment (rootulp) [reply cips/cip-14.md:38]: I expect more messages to be added to this list in the future but IMO this list should be expanded via the CIP process rather than on-chain governance.

File: cips/cip-14.md
Comment (rootulp) [reply cips/cip-14.md:42]: can do!

Comment (jcstein) [reply cips/cip-14.md:42]: tagging authors @womensrights @asalzmann @sampocs for review

**Review by jcstein — APPROVED** [review]
LGTM, but will wait on review from @womensrights @asalzmann and @sampocs

**Comment by womensrights** [comment]
lgtm, I will submit a seperate pr linking to test cases in the ibc-go repo

**Review by sampocs — APPROVED** [review]
lgtm, thanks for updating!

**Pull request merged**
